core/src/main/scala/not/ogame/bots/OgameDriver.scala

```diff
@@ -10,7 +10,19 @@ case class Credentials(login: String, password: String, universeName: String, un
 
 trait OgameDriver[F[_]] {
   def login(): F[Unit]
-  def getFactories(planetId: String): F[PlanetFactories]
+
+  def getSuppliesLevels(planetId: String): F[SuppliesLevels]
 }
 
-case class PlanetFactories(metal: Int)
+case class SuppliesLevels(map: Map[SuppliesBuilding.Value, Int])
+
+object SuppliesBuilding extends Enumeration {
+  type SuppliesBuilding = Value
+  val METAL_MINE, CRYSTAL_MINE, DEUTERIUM_SYNTHESIZER, SOLAR_PLANT, METAL_STORAGE, CRYSTAL_STORAGE,
+  DEUTERIUM_STORAGE: SuppliesBuilding.Value = Value
+}
+
+object FacilityBuilding extends Enumeration {
```

Comment:
`Enumeration` is broken in scala 2. It will be fixed in scala 3 but for now it is better not to use it.
Replace it with `sealed trait` with `case object`s